playground/CosmosEndToEnd/CosmosEndToEnd.AppHost/Program.cs

```diff
@@ -9,12 +9,14 @@
                 .RunAsPreviewEmulator(e => e.WithDataExplorer());
 
 var db = cosmos.AddCosmosDatabase("db");
-var container = db.AddContainer("entries", "/id");
+var entries = db.AddContainer("entries", "/id");
+var users = db.AddContainer("users", "/id");
 
 builder.AddProject<Projects.CosmosEndToEnd_ApiService>("api")
        .WithExternalHttpEndpoints()
        .WithReference(db).WaitFor(db)
-       .WithReference(container).WaitFor(container);
+       .WithReference(entries).WaitFor(entries)
+       .WithReference(users).WaitFor(users);
```

Comment:
You really only need to `.WithReference(db).WaitFor(db);`. The ApiService doesn't use the `entries` and `users` references, so we can remove them here.